### PR TITLE
updating cached_datasets.json with cifar10

### PR DESCRIPTION
**Pull request by lcadalzo**
the config is already updated, and preprocessing has already been added. So this is just a simple addition to the json

### CODE REVIEW

**Review by ppark-twosixtech — APPROVED** [review]
LGTM, pulled the cached dataset successfully

> 2022-12-22 21:45:26  8s INFO     armory.scenarios.scenario:load_train_dataset:198 Loading train dataset cifar10 with kwargs {'batch_size': 64, 'epochs': 20, 'name': 'cifar10', 'shuffle_files': True, 'split': 'train'}
2022-12-22 21:45:28  9s INFO     armory.datasets.load:load:71 dataset cifar10 not found locally
2022-12-22 21:45:28  9s INFO     armory.datasets.download:download_file_from_s3:33 downloading S3 data file armory-public-data/datasets/cache/cifar10__3.0.2__cache.tar.gz
datasets/cache/cifar10__3.0.2__cache.tar.gz download:   0% 0.00/135M [00:00<?, ?B/s]
datasets/cache/cifar10__3.0.2__cache.tar.gz download:   0% 262k/135M [00:00<02:18, 974kB/s]
